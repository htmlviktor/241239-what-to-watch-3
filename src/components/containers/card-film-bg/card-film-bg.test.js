import React from 'react';
import renderer from 'react-test-renderer';
import CardFilmBg from "./card-film-bg";


it(`CardFilmBg snapshot test`, () => {
  const tree = renderer
    .create(<CardFilmBg />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
