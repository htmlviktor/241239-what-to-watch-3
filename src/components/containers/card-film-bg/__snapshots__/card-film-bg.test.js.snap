// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardFilmBg snapshot test 1`] = `
<div
  className="movie-card__bg"
>
  <img
    alt="The Grand Budapest Hotel"
    src="img/bg-the-grand-budapest-hotel.jpg"
  />
</div>
`;
